scripts/sct_apply_transfo.py

```diff
@@ -176,8 +176,7 @@ def apply(self):
             isLastAffine = True
 
         # check if destination file is 3d
-        if not sct.check_if_3d(fname_dest):
-            sct.printv('ERROR: Destination data must be 3d')
+        sct.check_dim(fname_dest, dim_lst=[3])
```

Comment:
This line causes a fail of Travis. It seems like the `dmri` image is 4D. Previously: only a error message was displayed. Now: we do a system exit.

Reply:
Well... if it’s an error, Travis should fail. If the process continues, it should be a warning. Does the process continue here?

Reply:
Before: Error message printed, process continued.
Now: process stops.

Reply:
Ok so I would just replace the ERROR with a warning, and use ‘warning’ for the last argument

Reply:
Sorry that won’t work. So let’s just skip the line

Reply:
Probably some legacy code...

Reply:
Just to confirm: you are recommending to remove this `check_dim` from `sct_apply_transfo`, correct?

Reply:
Yes indeed